tests/accessor/host_accessor_constructors_core.cpp

```diff
@@ -23,11 +23,7 @@ namespace host_accessor_constructors_core {
 DISABLED_FOR_TEST_CASE(hipSYCL, ComputeCpp, DPCPP)
 ("sycl::host_accessor constructors. core types", "[accessor]")({
   using namespace host_accessor_constructors;
-#ifndef SYCL_CTS_ENABLE_FULL_CONFORMANCE
-  const auto types = get_lightweight_type_pack();
-#else
-  const auto types = get_full_conformance_type_pack();
-#endif  // SYCL_CTS_ENABLE_FULL_CONFORMANCE
+const auto types = get_conformance_type_pack();
```

Comment:
As you adding `get_conformance_type_pack` in this PR, can you please apply this change to `local_accessor` tests?

Reply:
thanks to this tip :) 
 have done this in this commit: 654aa4f1990ae6eccd22f5a9ed74dc553d8f6e86